ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -301,24 +305,50 @@ const PANELS = PANEL_DETAILS.map(panelDetails)
 const STAFF_PANELS = STAFF_PANEL_DETAILS.map(panelDetails)
 
 class VariantSearchFormContent extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { activeIndex: [] }
+  }
+
+  expandAll = () => {
+    this.setState({ activeIndex: [0, 1, 2, 3, 4, 5] })
+  }
+
+  collapseAll = () => {
+    this.setState({ activeIndex: [] })
+  }
+
+  handleTitleClick = (e, itemProps) => {
+    const { index } = itemProps
+    const { activeIndex } = this.state
+    const newIndex = activeIndex.indexOf(index) === -1 ? [...activeIndex, index] : activeIndex.filter(item => item !== index)
+
+    this.setState({ activeIndex: newIndex })
+  }
+
   render() {
     return (
       <div>
         <ProjectFamiliesField />
         <VerticalSpacer height={10} />
         <InlineHeader content="Saved Search:" />
         {configuredField(SAVED_SEARCH_FIELD)}
+        <ExpandCollapseCategoryContainer>
+          <ButtonLink onClick={this.expandAll}>Expand All &nbsp;<Icon name="plus" /></ButtonLink>
+          <b>| &nbsp;&nbsp;</b>
+          <ButtonLink onClick={this.collapseAll}>Collapse All &nbsp;<Icon name="minus" /></ButtonLink>
+        </ExpandCollapseCategoryContainer>
         <VerticalSpacer height={10} />
         <FormSection name="search">
-          <Accordion fluid panels={this.props.user.isStaff ? STAFF_PANELS : PANELS} />
+          <Accordion fluid panels={this.props.user.isStaff ? STAFF_PANELS : PANELS} exclusive={false} activeIndex={this.state.activeIndex} onTitleClick={this.handleTitleClick} />
         </FormSection>
       </div>
     )
   }
 
   shouldComponentUpdate() {
     // Form content does not use passed props, so should never re-render on prop update
-    return false
+    return true
```

Comment:
So what this function is doing is preventing react from unnecessarily re-rendering any of the internal content when props change. Changing this to true will cause it to ALWAYS re-render, even if none of the state or props have changed, which would be pretty bad for performance. Instead, we want this to return true if the state has changed, and return false otherwise, entirely ignoring the props. 

You can read more about this method here:
https://reactjs.org/docs/react-component.html#shouldcomponentupdate

I would recommend you try to figure out on your own how you can rewrite this function so it adds the behavior you need (actually updating when the state is changed) without getting rid of the previous behavior we want to keep (not updating when the props change). If you do find yourself struggling with this, feel free to ping me and I'd be happy to work through it with you